### PR TITLE
Am update new admin form

### PR DESCRIPTION
**Pull request by atulyamiglani**
https://app.clickup.com/t/23yhk2b

Adds first name and last name fields to the AddAdminPage form. 
Adds the same fields to the API request method.

### CODE REVIEW

**Comment by rymaju** [comment]
Looks like the weird .icloud file got uploaded too ‼️

File: cypress/integration/add-admin.spec.ts [hunk 1]
```diff
@@ -1,23 +1,28 @@
...
+    const email = 'something@something.com';
+    cy.intercept('POST', '/user', {
+      firstName: 'something',
+      lastName: 'somethingElse',
+      email,
+      role: 'admin',
+    }).as('createAdmin');
+    cy.loginAsResearcher();
+    cy.visit('/private/add-new-admin');
+    cy.get('#admin-email').click();
```
Comment (rymaju) [inline cypress/integration/add-admin.spec.ts:16]: ⛏️ a helper to click and type would reduce some code duplication here

File: src/pages/researcherLandingPage/ResearcherLandingPage.test.tsx [hunk 2]
```diff
@@ -10,7 +10,9 @@ jest.mock('../../api/apiClient');
 describe('Landing', () => {
   test('renders researcher landing page list of admins', async () => {
     apiClient.getAdmins = jest.fn(() =>
-      Promise.resolve([{ id: 1, email: 'test@test.com', role: 'admin' }]),
+      Promise.resolve([
+        { id: 1, firstName: 'first', lastName: 'last', email: 'test@test.com', role: 'admin' },
```
Comment (rymaju) [inline src/pages/researcherLandingPage/ResearcherLandingPage.test.tsx:14]: ⛏️ Would prefer to use the ADMIN enum constant

**Review by rymaju — APPROVED** [review]
Looks good once you remove the icloud file, left some small nitpicks.

Overall great work 💯

File: src/api/contracts/createUser.test.pact.js [hunk 3]
```diff
@@ -1,47 +0,0 @@
-// eslint-disable-next-line import/no-extraneous-dependencies
-import * as Pact from '@pact-foundation/pact';
-import { ApiClient } from '../apiClient';
-
-describe('POST /user', () => {
```
Comment (onb6) [inline src/api/contracts/createUser.test.pact.js:5]: Looks like the file was deleted?

**Review by onb6 — APPROVED** [review]

**Pull request merged**